resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -47,9 +47,19 @@ rm -f "$PIDFILE"
 
 init_restart_file "$restartfile" || exit $?
 
+CLASSPATH=${INSTALL_DIR}/puppet-server-release.jar
+
+if [ -e "${INSTALL_DIR}/cli/apps/cli-defaults.sh" ]; then
```

Comment:
Agreed. Another option I don't like very much is changing the `cli-app.erb` script to be aware of `cli-defaults.sh`